config/environments/production.rb

```diff
@@ -64,6 +64,8 @@
 
   config.action_mailer.perform_caching = false
 
+  # config.action_mailer.default_url_options = {host:'mydomain.com'}
+
```

Comment:
Why you need it?

Reply:
I will need this when I work not with localhost (next point of pdp)